docs/topics/wasm/wasm-js-interop.md

```diff
@@ -359,7 +359,57 @@ external fun <T : JsAny> processData(data: JsArray<T>): T
 
 ## Exception handling
 
-The Kotlin `try-catch` expression can't catch JavaScript exceptions.
+Since Kotlin 2.0.0 the Kotlin `try-catch` expression can catch JavaScript exceptions.
+However, accessing specific details about the thrown value in Kotlin/Wasm isn’t possible by default.
+
+Starting from Kotlin 2.1.0, you can configure `JsException` to include the original error message and stack trace from JavaScript by enabling the `-Xwasm-attach-js-exception` compiler flag.
+
+To do so, add the following to your `build.gradle.kts` file:
+
+```kotlin
+kotlin {
+    wasmJs {
+        compilerOptions {
+            freeCompilerArgs.add("-Xwasm-attach-js-exception")
+        }
+    }
+}
+```
+
+This behavior depends on the `WebAssembly.JSTag` API, which is only available in certain browsers:
+
+* Chrome: Supported from version 115
+* Firefox: Supported from version 129
+* Safari: Not yet supported
```

Comment:
It would be great if we link to a page/issue where the progress on this support is tracked, if it exists. Something link: https://webassembly.org/features/

Reply:
Unfortunately, I cannot find anything :(